### PR TITLE
feat: loki and promtail upgrade

### PR DESCRIPTION
**Pull request by srodenhuis**
This PR brings an upgrade for Loki and Promtail to version 2.8.2

 Summary:

- Compactor, Distributor, Gateway, Ingester, Querier and Querier frontend are by default deployed
- Storage now by default configured to use in-cluster Minio for storage
- API now only supports s3 (compatible), azure blob and gcs. More can be added if required
- No changes in the multi-tenant setup
- Authentication for multi-tenancy is moved to the  loki query frontend
- All components are setup for scaling and scaling is now in API schema

## Checklist

- [ ] Architecture Design Records have been added as `adr/*.md` and appended to list in `adr/_index.md`, if applicable.
- [x] The `values-schema.yaml` file and `test/**` fixtures have been updated to reflect code changes, if applicable.
- [x] The OpenApi Schema from redkubes/otomi-api project is compatible with definitions from `values-schema.yaml` file, if applicable.
- [x] Helm releases are meeting otomi's baseline security policies, if applicable.
- [ ] Helm chart and helmfile changes are tested against upgrade scenario, if applicable.

### CODE REVIEW

**Comment by Ani1357** [comment]
update path from v0.21.4 in both minikubes and DO fails at the test job with the following error
image

**Comment by srodenhuis** [comment]
@Ani1357 fixtures for tests have been adjusted and the test now passes.

**Review by Ani1357 — APPROVED** [review]
successfully tested the upgrade from v0.21.4

File: values-schema.yaml [hunk 1]
```diff
@@ -2316,40 +2353,43 @@ properties:
...
-                type: object
-              cassandra:
-                type: object
               gcs:
-                type: object
-              swift:
-                type: object
-              local:
-                type: object
-              storageType:
```
Comment (j-zimnowoda) [inline values-schema.yaml:2342]: Here things go wrong: schema has been changed but there are no migrations defined in the `values-changes.yaml` file

**Pull request merged**